code/game/objects/structures/crates_lockers/closets/secure/security.dm

```diff
@@ -42,6 +42,8 @@
 	new /obj/item/door_remote/civillian(src)
 	new /obj/item/circuitboard/machine/techfab/department/service(src)
 	new /obj/item/storage/photo_album/HoP(src)
+	new /obj/item/card/id/departmental_budget/srv (src)
+	new /obj/item/card/id/departmental_budget/civ (src)
```

Comment:
Adding a civilian budget is... interesting for sure

Reply:
Bup pointed it out when some jobs pay under civil, not service